### ISSUE TITLE
Cannot install 0.2

### ISSUE THREAD

**Issue by andig** [containrrr/shoutrrr#87]
```
❯ go get github.com/containrrr/shoutrrr@v0.2  
go get github.com/containrrr/shoutrrr@v0.2: no matching versions for query "v0.2"
```

I may be wrong, but imho go semver requires 0.2.0 as version.

**Comment by simskij** [comment]
Hey @andig! Thanks for letting us know! We'll switch over to the go mod recommended format of semver with the next release.

**Comment by piksel** [comment]
Yeah, I explicitly put `v0.3.0` as the milestone name for this reason!

**Comment by simskij** [comment]
Will that be enough? I'm wondering whether we also have to drop the leading `v`? I'll make sure to look that up before we do the next release.

**Comment by simskij** [comment]
Solved with `v0.3.0` which we just released 🎉
